openapi.yaml

```diff
@@ -561,7 +578,7 @@ components:
           type: boolean
         warning_type:
           type: ["string", "null"]
-          enum: *warning-types
+          "$ref": "#/components/schemas/WarningTypes"
```

Comment:
Does the type in the component override the type set here?

Reply:
Ah, yes. There is an error. The ref overwrites the  type which is set here. I will fix it.